docs/references/react/use-user.mdx

```diff
@@ -1,49 +1,47 @@
 ---
 title: '`useUser()`'
-description: The useUser() hook is used to get the current user object and to update the user's data on the client-side.
+description: Access and manage the current user's data in your React application with Clerk's useUser() hook.
 ---
 
-The `useUser()` hook is a convenient way to access the current user's [`User`](/docs/references/javascript/user/user) object, which holds all of the information for a single user of your application and provides a set of methods to manage their account. This hook also provides boolean values to check if the user is signed in and if Clerk has loaded and initialized.
+The `useUser()` hook provides access to the current user's [`User`](/docs/references/javascript/user/user) object, which contains all the data for a single user in your application and provides methods to manage their account. This hook also allows you to check if the user is signed in and if Clerk has loaded and initialized.
 
-## `useUser()` returns
+## Returns
 
 <Properties>
-  - `isSignedIn`
+  - `isLoaded`
   - `boolean`
 
-  A boolean that returns `true` if the user is signed in.
+    A boolean that indicates whether Clerk has completed initialization. Initially `false`, becomes `true` once Clerk loads.
 
   ---
 
-  - `isLoaded`
+  - `isSignedIn`
   - `boolean`
 
-  A boolean that until Clerk loads and initializes, will be set to `false`. Once Clerk loads, `isLoaded` will be set to `true`.
+  A boolean that returns `true` if the user is signed in.
 
   ---
 
   - `user`
   - [`User`](/docs/references/javascript/user/user)
```

Comment:
that backslash might not be needed - i'd check locally, see how its rendered with and without

```suggestion
  - <code>[User](/docs/references/javascript/user/user) \| null</code>
```